static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -680,10 +683,11 @@ define([
          * Module
          */
         dfp = {
-            init:        init,
-            addSlot:     addSlot,
-            refreshSlot: refreshSlot,
-            getSlots:    getSlots,
+            init:           init,
+            addSlot:        addSlot,
+            refreshSlot:    refreshSlot,
+            getSlots:       getSlots,
+            shouldLazyLoad: shouldLazyLoad,
```

Comment:
Given that this is a private method, I think we should either name it as such with a dangling underscore or extract it to a different file, like dfp-helpers.js. But it should not look like an ordinary public interface.